clients/java/src/main/java/io/camunda/client/impl/CamundaClientBuilderImpl.java

```diff
@@ -208,6 +211,11 @@ public List<ClientInterceptor> getInterceptors() {
   }
 
   @Override
+  public List<AsyncExecChainHandler> getChainHandlers() {
+    return chainHandlers;
+  }
+
+    @Override
```

Comment:
🔧 Indentation is a bit off here, same in other places.

Reply:
Also, imports seem to be a bit off here and in other classes, the Java checks are failing due to this.

Reply:
👍 I opened the `clients/java` module as a separate project in IntelliJ since I was having some issues with dependencies, and it seems IntelliJ doesn't pick up some of the formatting rules. I'll fix them.